docs/rule/require-aria-activedescendant-tabindex.md

```diff
@@ -0,0 +1,34 @@
+# require-aria-activedescendant-tabindex
+
+This rule requires all noninteractive HTML elements using the aria-activedescendant attribute to declare a tabindex of zero.
+
+The `aria-activedescendant` attribute identifies the active descendant element of a composite widget, textbox, group, or application with document focus. This attribute is placed on the container element of the input control, and its value is set to the ID of the active child element. This allows screen readers to communicate information about the currently active element as if it has focus, while actual focus of the DOM remains on the container element.
+
+Elements with `aria-activedescendant` must have a tabindex of zero in order to support keyboard navigation. Besides interactive elements, which are inherently keyboard-focusable, elements using the `aria-activedescendant` attribute must declare a `tabIndex` of zero with the `tabIndex` attribute.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<div aria-activedescendant="some-id" />
+<div aria-activedescendant="some-id" tabindex="-1" />
+<input aria-activedescendant={{some-id}} tabindex="-1" />
+```
+
+This rule **allows** the following:
+
+```hbs
+<CustomComponent />
+<CustomComponent aria-activedescendant={{some-id}} />
+<CustomComponent aria-activedescendant={{some-id}} tabindex={{0}} />
+<div aria-activedescendant="some-id" tabindex="0" />
+<input />
+<input aria-activedescendant={{some-id}} />
+<input aria-activedescendant={{some-id}} tabindex={{0}} />
+```
+
+## References
```

Comment:
Can we link to the other rule too? https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/aria-activedescendant-has-tabindex.md